debian/control

```diff
@@ -1,16 +1,17 @@
 Source: revpi-modbus
 Section: contrib/electronics
 Priority: optional
-Build-Depends: debhelper-compat (= 12),
-               libmodbus-dev (>= 3.1.6),
-               libjson-c-dev,
-               cmake
+Build-Depends:
```

Comment:
I did no see it before, but usually we are using in other repos more information like
'''
Maintainer: KUNBUS GmbH <support@kunbus.com>
Rules-Requires-Root:  no
'''
And build depends should be placed after vcs-git.
We usually use a "extra commit" for reorder this...

Reply:
and also the dependencies can be modified... Unfortunately I can't mark it.

Reply:
`Depends: ${shlibs:Depends}, ${misc:Depends}`

```
Depends:
  ${shlibs:Depends},
  ${misc:Depends}
```